### ISSUE TITLE
Building: could not determine kind of name for C.gtk_layer_set_keyboard_mode

### ISSUE THREAD

**Issue by jspkay** [nwg-piotr/nwg-bar#37]
I'm trying to build the project on my machine. Unfortunatly the build fails and I have no clue why. 

OS: Debian 11 

```
$ go build -v -o bin/nwg-bar -tags "glib_2_66" .
github.com/dlasky/gotk3-layershell/layershell
# github.com/dlasky/gotk3-layershell/layershell
../../go/pkg/mod/github.com/dlasky/gotk3-layershell@v0.0.0-20240515133811-5c5115f0d774/layershell/layer.go:100:35: could not determine kind of name for C.GtkLayerShellKeyboardMode
../../go/pkg/mod/github.com/dlasky/gotk3-layershell@v0.0.0-20240515133811-5c5115f0d774/layershell/layer.go:100:2: could not determine kind of name for C.gtk_layer_set_keyboard_mode
make: *** [Makefile:13: build] Error 1
```

**Comment by nwg-piotr** [comment]
Looks like the gtk-layer-shell version too old.

**Comment by jspkay** [comment]
Thank you so much! I solved installing v0.8.0
